dsp/starchgen.py

```diff
@@ -32,6 +32,11 @@
                features = ['neon'],
                test_function = 'cpu_supports_armv7_neon_vfpv4',
                alignment = 16)
+gen.add_flavor(name = 'armv8_a',
```

Comment:
Does this need a `-march=...` flag?

I'd somewhat expect the NEON intrinsics code to work in aarch64 mode - should this set features=neon?